app/index.js

```diff
@@ -152,31 +154,27 @@ module.exports = yeoman.generators.Base.extend({
 			{
 				buildFormat: this.buildFormat,
 				componentName: this.componentName,
-				templateLanguage: this.templateLanguage
+				templateLanguage: this.templateLanguage,
+				kebabCaseName: this.kebabCaseName
 			}
 		);
 		this.fs.copyTpl(
-			this.templatePath('test/_Boilerplate.js'), this.destinationPath('test/' + this.componentName + '.js'),
+			this.templatePath('__tests__/_Boilerplate.js'), this.destinationPath('__tests__/' + this.componentName + '.js'),
```

Comment:
Not exactly. It's just a configuration option for Karma. I thought to keep the same folder name to avoid the condition of the test environment. This name is the default for Jest and for Karma we always have to specify the test folder. So, made more sense for me to use `__tests__`. Does it make sense?